### PR TITLE
nvidia: update to 560.35.03

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- nvidia: update to 560.35.03

Package(s) Affected
-------------------

- nvidia: 560.35.03

Security Update?
----------------

No

Build Order
-----------

```
#buildit nvidia
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

### CODE REVIEW

File: runtime-display/nvidia/autobuild/build [hunk 1]
```diff
@@ -106,7 +109,7 @@ cd ..
 abinfo "NVIDIA blacklist for Nouveau..."
 install -dm 755 "$PKGDIR"/usr/{lib/modprobe.d,src}
 cp -dr --no-preserve='ownership' kernel "$PKGDIR"/usr/src/nvidia-$PKGVER
-echo 'blacklist nouveau' > "$PKGDIR"/usr/lib/modprobe.d/nvidia.conf
+echo 'blacklist nouveau' >"$PKGDIR"/usr/lib/modprobe.d/nvidia.conf
```
Comment (MingcongBai) [inline runtime-display/nvidia/autobuild/build]: Drop this change.

File: runtime-display/nvidia/autobuild/build [hunk 2]
```diff
@@ -61,24 +61,27 @@ sanity_check() {
 NV_ARCH="undefined"
 
 case "$ABHOST" in
-    amd64)
```
Comment (MingcongBai) [inline runtime-display/nvidia/autobuild/build]: Why no indent?

File: runtime-display/nvidia/autobuild/build [hunk 3]
```diff
@@ -273,11 +292,11 @@ install_for_all 755 libnvidia-gtk3.so.$PKGVER "$PKGDIR"/usr/lib
 
 abinfo "Power management services"
 for i in suspend hibernate resume; do
-    install_for_all 644 systemd/system/nvidia-${i}.service \
+    install -Dvm644 systemd/system/nvidia-${i}.service \
```
Comment (MingcongBai) [inline runtime-display/nvidia/autobuild/build]: Hmm?

File: runtime-display/nvidia/autobuild/build [hunk 4]
```diff
@@ -319,3 +338,9 @@ if [[ $NV_ARCH == "x86_64" ]]; then
     source "${SRCDIR}/autobuild/build-optenv32"
     popd
 fi
+
+# install -Dm644 "${SRCDIR}"/systemd-homed-override.conf "${PKGDIR}"/usr/lib/systemd/system/systemd-homed.service.d/10-nvidia-no-freeze-session.conf
```
Comment (MingcongBai) [inline runtime-display/nvidia/autobuild/build]: What are these for?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by xtexx** [comment]
LGTM on AMD64
```
xtex@xtex1 ~ (main)> nvidia-smi
Mon Aug 26 10:08:48 2024       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 560.35.03              Driver Version: 560.35.03      CUDA Version: 12.6     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce GTX 750 Ti      On  |   00000000:01:00.0  On |                  N/A |
| 33%   43C    P0              2W /   38W |     618MiB /   2048MiB |     10%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
```
Tested glxgears, vulkaninfo, nvidia-smi, gltestperf

File: runtime-display/nvidia/autobuild/build (hunk 3 above)
Comment (xtexx) [reply runtime-display/nvidia/autobuild/build]: `install_for_all` will install files to `/usr/lib/systemd/system/nvidia-${i}.service/nvidia-${i}.service`

File: runtime-display/nvidia/autobuild/build (hunk 4 above)
Comment (xtexx) [reply runtime-display/nvidia/autobuild/build]: Removed. They was for testing.

File: runtime-display/nvidia/autobuild/build (hunk 3 above)
Comment (MingcongBai) [reply runtime-display/nvidia/autobuild/build]: > `install_for_all` will install files to `/usr/lib/systemd/system/nvidia-${i}.service/nvidia-${i}.service`

Which means that the scripting had created that directory before hand?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by OriginCode** [comment]
LGTM on amd64

**Review by OriginCode — APPROVED** [review]

**Pull request merged**